src/Connect4.rdef

```diff
@@ -0,0 +1,72 @@
+
+resource large_icon array {
+	$"1212121212121212121212121212121212121212121212121212121212121204"
+	$"12122121211212122A2A2A121212212121121212212121121212212121121204"
+	$"122121212121122A2A2A2A2A1221212121211221212121211221212121211204"
+	$"122121212121122A2A2A2A2A1221212121211221212121211221212121211204"
+	$"122121212121122A2A2A2A2A1221212121211221212121211221212121211204"
+	$"12122121211212122A2A2A121212212121121212212121121212212121121204"
+	$"1212121212121212121212121212121212121212121212121212121212121204"
+	$"12122121211212122121211212122121211212122A2A2A121212212121121204"
+	$"122121212121122121212121122121212121122A2A2A2A2A1221212121211204"
+	$"122121212121122121212121122121212121122A2A2A2A2A1221212121211204"
+	$"122121212121122121212121122121212121122A2A2A2A2A1221212121211204"
+	$"12122121211212122121211212122121211212122A2A2A121212212121121204"
+	$"1212121212121212121212121212121212121212121212121212121212121204"
+	$"12122A2A2A1212122121211212122A2A2A121212212121121212212121121204"
+	$"122A2A2A2A2A122121212121122A2A2A2A2A1221212121211221212121211204"
+	$"122A2A2A2A2A122121212121122A2A2A2A2A1221212121211221212121211204"
+	$"122A2A2A2A2A122121212121122A2A2A2A2A1221212121211221212121211204"
+	$"12122A2A2A1212122121211212122A2A2A121212212121121212212121121204"
+	$"1212121212121212121212121212121212121212121212121212121212121204"
+	$"12122121211212122A2A2A1212122121211212122121211212122A2A2A121204"
+	$"122121212121122A2A2A2A2A122121212121122121212121122A2A2A2A2A1204"
+	$"122121212121122A2A2A2A2A122121212121122121212121122A2A2A2A2A1204"
+	$"122121212121122A2A2A2A2A122121212121122121212121122A2A2A2A2A1204"
+	$"12122121211212122A2A2A1212122121211212122121211212122A2A2A121204"
+	$"1212121212121212121212121212121212121212121212121212121212121204"
+	$"12122121211212122121211212122A2A2A121212212121121212212121121204"
+	$"122121212121122121212121122A2A2A2A2A1221212121211221212121211204"
+	$"122121212121122121212121122A2A2A2A2A1221212121211221212121211204"
+	$"122121212121122121212121122A2A2A2A2A1221212121211221212121211204"
+	$"12122121211212122121211212122A2A2A121212212121121212212121121204"
+	$"1212121212121212121212121212121212121212121212121212121212121204"
+	$"0404040404040404040404040404040404040404040404040404040404040404"
+};
+
+resource mini_icon array {
+	$"102010101020101010201010102A1003"
+	$"2020201020202010202020102A2A2A03"
+	$"102010101020101010201010102A1003"
+	$"10101010101010101010101010101003"
+	$"102A101010201010102A101010201003"
+	$"2A2A2A10202020102A2A2A1020202003"
+	$"102A101010201010102A101010201003"
+	$"10101010101010101010101010101003"
+	$"10201010102A10101020101010201003"
+	$"202020102A2A2A102020201020202003"
+	$"10201010102A10101020101010201003"
+	$"10101010101010101010101010101003"
+	$"1020101010201010102A101010201003"
+	$"20202010202020102A2A2A1020202003"
+	$"1020101010201010102A101010201003"
+	$"03030303030303030303030303030303"
+};
+
+resource app_signature "application/x-vnd.Connect-4";
```

Comment:
Please move everything from line 58ff up to the top of the file.

Reply:
@humdingerb So I just put everything from line 58 and after to the beginning of the file like this?